src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -63,6 +73,50 @@ export class InputControlProvider {
         this._setInputControlDataSourceInputs();
     }
 
+    private async setInputControlValueFromRepoAnalysisResult(inputControl: InputControl): Promise<void> {
+        let repoAnalysisSettingKey = inputControl.getPropertyValue(this.repoAnalysisSettingKey);
+        if (this._repoAnalysisSettingInUse !== -1) {
+            if (this._repoAnalysisSettings.length === 0 || !this._repoAnalysisSettings[this._repoAnalysisSettingInUse].settings[repoAnalysisSettingKey]) {
```

Comment:
should we not be even inside setInputControlValueFromRepoAnalysisResult function if `this._repoAnalysisSettings.length === 0`. this will keep the code cleaner

Reply:
Done